docs/customer/routing.md

```diff
@@ -116,6 +116,8 @@ Used for troubleshooting, you can remove carriers from a route and run a quick t
 
 + **Lock** (Allow): One or more rate cards from the list of available providers.
 + **Exclude** (Deny): Exclude access to one or more rate cards in the list of available providers.
++ **Redial Max Count**: This a smart limitation feature that allows the carrier to restrict the maximum number of times their customers can redial. After reaching this limit, customers must wait for a certain amount of time defined by the **Redial Max Period**. For example, you select 5 for this field, it means your customer can dial only 5 times.
++ **Redial Max Period**: Select the time-period for which you can allow the customer to redial. For example, if you select Redial Max Count as 10 and Redial Max Period as 6000 seconds, this means your customer can redial maximum 10 times within 6000 seconds.
```

Comment:
Redial Max Period" refers to the duration of time during which a customer is restricted from making further redial attempts to the same number after reaching the maximum redial count.
No need for example